lib/oga/xml/document.rb

```diff
@@ -57,6 +57,12 @@ def root_node
         self
       end
 
+      # Returns the top level element in the document.
+
```

Comment:
In more concrete terms, this means for users of other libraries:

    doc << element

Has to be replaced by this:

    doc.children.last.children << element

Or, more verbose, but at least readable:

    root = doc.children.last
    root.children << element